ckanext/saml2auth/plugin.py

```diff
@@ -48,8 +49,8 @@ class Saml2AuthPlugin(plugins.SingletonPlugin):
 
     def get_helpers(self):
         return {
-            'is_default_login_enabled':
-                h.is_default_login_enabled
+            'is_default_login_enabled': h.is_default_login_enabled,
+            'get_saml2auth_login_button_text': helpers.get_saml2auth_login_button_text,
```

Comment:
```suggestion
            'get_saml2auth_login_button_text': h.get_saml2auth_login_button_text,
```
We have two imports for the same thing.
Choose one

```
from ckanext.saml2auth import helpers as h
OR
from ckanext.saml2auth import helpers
```